packages/fx-core/resource/package.nls.json

```diff
@@ -522,6 +522,18 @@
   "core.common.ErrorFetchApiSpec": "Your OpenAPI description document should be accessible without authentication, otherwise download and start from a local copy.",
   "core.common.SendingApiRequest": "Sending API request: %s. Request body: %s",
   "core.common.ReceiveApiResponse": "Received API response: %s.",
+  "core.envFunc.unsupportedFile.errorLog": "\"%s\" is not a valid file. Supported format: %s.",
+  "core.envFunc.unsupportedFile.errorMessage": "Invalid file. Check output for details.",
+  "core.envFunc.unsupportedFile.errorMessage.vsc": "Invalid file. Check [Output panel](command:fx-extension.showOutputChannel) for details.",
+  "core.envFunc.unsupportedFunction.errorLog": "\"%s\" is not a valid function. Supported function: %s.",
+  "core.envFunc.unsupportedFunction.errorMessage": "Invalid function. Check output for details.",
+  "core.envFunc.unsupportedFunction.errorMessage.vsc": "Invalid function. Check [Output panel](command:fx-extension.showOutputChannel) for details.",
+  "core.envFunc.invalidFunctionParameter.errorLog": "The parameter \"%s\" of function \"%s\" is invalid. Please provide a valid file path wrapped by '' or an environment variable name in \"${{}}\" format.",
+  "core.envFunc.invalidFunctionParameter.errorMessage": "Invalid parameter of function \"%s\". Check output for details.",
+  "core.envFunc.invalidFunctionParameter.errorMessage.vsc": "Invalid parameter of function \"%s\". Check [Output panel](command:fx-extension.showOutputChannel) for details.",
+  "core.envFunc.readFile.errorLog": "Failed to read from \"%s\" due to \"%s\".",
+  "core.envFunc.readFile.errorMessage": "Failed to read from \"%s\". Check output for details.",
+  "core.envFunc.readFile.errorMessage.vsc": "Failed to read from \"%s\". Check [Output panel](command:fx-extension.showOutputChannel) for details.",
```

Comment:
Proposed: "Unable to read from \"%s\". Check [Output panel](command:fx-extension.showOutputChannel) for details."